execution_workflows/Roar/README.md

```diff
@@ -0,0 +1,47 @@
+# Roar
+Roar detects the potential Alternative Polyadenylation (APA) events in the downstream 3'-UTR of a gene among two different biological conditions.
+
+# Installation
+Roar can be installed from GitHub & BiocManager
+
+* GitHub: https://github.com/vodkatad/roar
+* BiocBioconductor: https://bioconductor.org/packages/release/bioc/html/roar.html
+
+# User manual
+Documentation for Roar is available at:
+
+* Tutorial: https://bioconductor.org/packages/release/bioc/vignettes/roar/inst/doc/roar.pdf
+* User Manual: https://bioconductor.org/packages/release/bioc/manuals/roar/man/roar.pdf
+
+# Environments
+
+**A Docker image still needs to be made for this**. All the needed packages and tools needed to run this are present in the conda `.yaml`, except for Roar itself which can be installed from GitHub/BioConductor (above).
```

Comment:
If the docker file you mention (link: https://quay.io/repository/biocontainers/bioconductor-roar?tab=tags) is up-to-date and also supported, then using that Docker image should be sufficient.